evap/settings.py

```diff
@@ -355,6 +368,25 @@
     "Enhances vibrant academic programs",
 ]
 
+
+### OpenID Login
+# replace 'example.com', OIDC_RP_CLIENT_ID and OIDC_RP_CLIENT_SECRET with real values in localsettings when activating
+ACTIVATE_OPEN_ID_LOGIN = False
+OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS = 60 * 60 * 24 * 7  # one week
+OIDC_USERNAME_ALGO = 'evap.evaluation.auth.generate_username_from_email'
```

Comment:
You asked whether to keep this - personally, I'd prefer to remove it. Right now, if user creation code of the OIDC plugin runs unexpectedly, it would probably add a user without cleaning the email correctly. If we make it an empty string, it should fail verbosely.